index.js

```diff
@@ -22,7 +22,13 @@ module.exports = function api(options) {
             password: arguments[2]
         };
     }
-
+    
+    //option: domain - in case the kudu api doesn't run on azurewebsites
+    var toplevelDomain = "scm.azurewebsites.net";
+    if(options.domain){
```

Comment:
This wont pass the ESLint spacing rules. Maybe the condition could be simplified to:

```
var domain = options.domain || "scm.azurewebsites.net";
```